app-server/app/src/main/scala/footprint/Filter.scala

```diff
@@ -0,0 +1,102 @@
+package com.azavea.rf.footprint
+
+import java.sql.Timestamp
+
+import com.lonelyplanet.akka.http.extensions.{PageRequest, Order}
+import geotrellis.slick.Projected
+import geotrellis.vector.{Point, Polygon}
+
+import com.azavea.rf.datamodel.latest.schema.tables._
+import com.azavea.rf.datamodel.driver.ExtendedPostgresDriver
+import com.azavea.rf.utils.queryparams._
+import com.azavea.rf.utils.UserErrorException
+
+object FootprintFilters {
+  import ExtendedPostgresDriver.api._
+
+  type FootprintsQuery = Query[Footprints, Footprints#TableElementType, Seq]
+
+  val datePart = SimpleFunction.binary[String, Option[Timestamp], Int]("date_part")
+
+  implicit class FootprintDefault[M, U, C[_]](footprints: FootprintsQuery) {
+    def filterByOrganization(orgParams: OrgQueryParameters): FootprintsQuery = {
+      footprints.filter{ footprint =>
+        orgParams.organization.map(footprint.organizationId === _)
+          .reduceLeftOption(_ || _).getOrElse(true:Rep[Boolean])
```

Comment:
Style comment, take from it what you will:

multiple transformations on a line is hard to read. 

```scala
footprints
  .filter { footprint =>
    orgParams.organization
      .map(footprint.organizationId === _)
      .reduceLeftOption(_ || _)
      .getOrElse(true:Rep[Boolean])
  }
```

reads much cleaner to me.